conf/routes

```diff
@@ -13,6 +13,8 @@ GET            /dashboard                                  controllers.Applicati
 GET            /about                                      controllers.Assets.at(path="/public", file="/static/about.html")
 GET            /editorialSupport                           controllers.Application.editorialSupport
 POST           /editorialSupport                           controllers.Application.updateEditorialSupport
+GET            /subscriptions                              controllers.Notifications.subscriptions
+POST           /subscriptions                              controllers.Notifications.deleteSubscription
```

Comment:
are we missing a `DELETE` route?

Reply:
Irritatingly you can't do `method="DELETE"` in an HTML form. I've worked around this in the editorial support page stuff by adding a hidden input `action=UPDATE` or `action=DELETE` but right now we can only delete so didn't see any reason to add it.